package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Resolve AMD module names to File objects suitable to run in the Browser",
   "main": "dist/amd-resolver.js",
   "scripts": {
-    "prepublish": "browserify src/resolver.js -o dist/amd-resolver.js -s amd-resolver --dg false",
+    "prepublish": "grunt compile-production",
```

Comment:
I've changed this to call `grunt`, so we may have to depend on `grunt-cli` now... I'm not sure how to best test whether we need this dependency or not.